package.json

```diff
@@ -23,13 +23,15 @@
     "testRegex": ".+\\.test\\.js$"
   },
   "peerDependencies": {
+    "prop-types": "^15.6.1",
```

Comment:
I think this should just go in `dependencies` - https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package

Reply:
Good catch! I made the change.